### PR TITLE
docs: adds Laravel docs.

### PR DESCRIPTION
**Pull request by tnolet**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer
- Adds Laravel docs
- Removes generic PHP placeholder

### CODE REVIEW

File: site/content/docs/traces-open-telemetry/instrumenting-code/laravel.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+
+// Bootstrap Laravel and handle the request...
+(require_once __DIR__.'/../bootstrap/app.php')
+    ->handleRequest(Request::capture());
+```
+
+## Step 3: Start your app with the instrumentation
+
+First, make sure to switch on the **Basic HTTP Instrumentation**. This will add the necessary headers to your HTTP requests.
```
Comment (MariadeAnton) [inline site/content/docs/traces-open-telemetry/instrumenting-code/laravel.md:166]:
```suggestion

```
we no longer need to flip instrumentation

File: site/content/docs/traces-open-telemetry/instrumenting-code/laravel.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+![Checkly basic OTEL http instrumentation](/docs/images/otel/otel_basic_instrumentation.png)
```
Comment (MariadeAnton) [inline site/content/docs/traces-open-telemetry/instrumenting-code/laravel.md:168]:
```suggestion
```

File: site/content/docs/traces-open-telemetry/instrumenting-code/laravel.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+Then, toggle on **Send Traces** and grab your OTel API key in the **OTel API keys** section of the [Open Telemetry Integration page in the Checkly app](https://app.checklyhq.com/settings/account/open-telemetry).
```
Comment (MariadeAnton) [inline site/content/docs/traces-open-telemetry/instrumenting-code/laravel.md:170]:
```suggestion
Toggle on **Send Traces** and grab your OTel API key in the **OTel API keys** section of the [Traces Integration page in the Checkly app](https://app.checklyhq.com/settings/account/open-telemetry).
```

**Review by MariadeAnton — COMMENTED** [review]
Other than the HTTP Instrumentation no longer needed, LGTM. Thank you so much, @tnolet !

**Comment by tnolet** [comment]
@MariadeAnton this is done

**Review by MariadeAnton — APPROVED** [review]
🚀

**Pull request merged**